package.json

```diff
@@ -1,27 +1,34 @@
 {
-  "name": "xrm-pipeline",
-  "version": "0.0.0",
-  "private": true,
-  "description": "Barnard pipeline with sensible defaults",
+  "name": "xrm-csvw-workflow",
+  "version": "1.0.0",
+  "description": "Barnard59 pipeline with sensible defaults",
+  "type": "module",
+  "main": "index.js",
   "scripts": {
-    "postinstall": "mkdirp output",
-    "start": "dotenv ./start.sh --"
+    "start": "mkdir -p output && barnard59 run -v --pipeline=http://example.org/pipeline/toFile pipelines/main.ttl",
+    "to-file": "mkdir -p output && barnard59 run -v --pipeline=http://example.org/pipeline/toFile pipelines/main.ttl",
+    "file-to-store": "barnard59 run -v --pipeline=http://example.org/pipeline/fileToStore pipelines/main.ttl --variable=ENDPOINT_URL --variable=ENDPOINT_USER --variable=ENDPOINT_PASSWORD",
+    "file-to-store-dev": "barnard59 run -v --pipeline=http://example.org/pipeline/fileToStore pipelines/main.ttl",
+    "to-store-dev": "barnard59 run -v --pipeline=http://example.org/pipeline/toStore pipelines/main.ttl",
+    "fetch": "barnard59 run -v --pipeline=http://example.org/pipeline/fetch pipelines/fetch.ttl --variable=SSH_HOST --variable=SSH_PORT --variable=SSH_USER --variable=SSH_PRIVATE_KEY",
+    "fetch-dev": "barnard59 run -v --pipeline=http://example.org/pipeline/fetch pipelines/fetch.ttl",
```

Comment:
I'm not sure if the fetch pipeline should be part of the template. It's specific to ssh. `pipelines/fetch.ttl` could be also removed.

Reply:
I was debating about it. We might say that fetch is only part of our template in GitLab but not for this public one on GitHub

Reply:
Not sure about GitLab, but I'm for removing it here. Less files -> less confusing -> higher acceptance.

Reply:
It's the internal one @ gov, will remove it here.

Reply:
Removed in https://github.com/zazuko/xrm-csvw-workflow/pull/16/commits/aecaff3d46dacca7da8a3520038b344651c97e89